apps/fortuna/src/config.rs

```diff
@@ -158,3 +158,72 @@ pub struct EthereumConfig {
     /// The gas limit to use for entropy callback transactions.
     pub gas_limit: U256,
 }
+
+#[derive(Args, Clone, Debug)]
+#[command(next_help_heading = "Provider Config Options")]
+#[group(id = "ProviderConfig")]
+pub struct ProviderConfigOptions {
+    /// Path to a configuration file containing provider configuration for the list of supported blockchains
+    /// The commitments should be sorted in ascending order by their original_committed_sequence_number.
+    #[arg(long = "provider-config")]
+    #[arg(env = "FORTUNA_PROVIDER_CONFIG")]
+    #[arg(default_value = "provider-config.yaml")]
+    pub provider_config: String,
```

Comment:
we should add a sample for this similar to what we have in config.sample.yaml

Reply:
agreed, on it